lib/locales/it.ts

```diff
@@ -432,6 +438,7 @@ export const locale: Locale = {
     our_lady_of_fatima: 'Beata Vergine Maria di Fatima',
     our_lady_of_guadalupe: 'Beata Vergine Maria di Guadalupe',
     our_lady_of_guadalupe_patroness_of_the_americas: 'Beata Vergine Maria di Guadalupe, Patrona delle Americhe',
+    our_lady_of_loreto: 'Beata Vergine Maria di Loreto', // src: missal_it_2020_ed3
```

Comment:
Just a nitpick: we might change this, however, everywhere I use `mr_${lang}_${year}_${edition}` format.

```suggestion
    our_lady_of_loreto: 'Beata Vergine Maria di Loreto', // src: mr_it_2020_ed3
```